kbase-extension/scss/partials/_custom.scss

```diff
@@ -339,23 +331,17 @@ p {
  * This hunk of CSS should remove it until we can better work it in to our design.
  */
 
-div.code_cell {
-  &:hover {
-    .run_this_cell,
-    div.input_prompt.run_this_cell {
-      visibility: hidden !important;
-    }
-  }
+div.run_this_cell {
```

Comment:
the icon in the 'run this cell' div was showing, so strong measures were required.